network.go

```diff
@@ -628,6 +638,17 @@ func NetworkOptionInternalNetwork() NetworkOption {
 	}
 }
 
+// NetworkOptionAttachable returns an option setter to set attachable for a network
+func NetworkOptionAttachable(attachable bool) NetworkOption {
+	return func(n *network) {
+		if n.generic == nil {
+			n.generic = make(map[string]interface{})
+		}
+		n.attachable = attachable
+		n.generic[netlabel.Attachable] = attachable
```

Comment:
Thanks @aboch. I was thinking maybe some drivers don't accept `attachable` but I don't find an example of it. I'll remove it.